src/js/charts/line.js

```diff
@@ -231,7 +243,8 @@
             legend_text.attr('fill', args.colors[which_line]);
           }
         } else {
-          legend_text.classed('mg-line' + (line_id) + '-legend-color', true);
+              legend_text.classed('mg-line' + (line_id) + '-legend-color', true);
+              legend_text.classed('mg-line' + '-legend-color', true);
         }
 
         mg_prevent_vertical_overlap(plot.legend_group.selectAll('.mg-line-legend text').nodes(), args);
```

Comment:
Remember to look at this file too as it isnt displayed by default because of high number of changes